k8s/aat/cluster-00/am/org-role-mapping-batch.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: am-org-role-mapping-batch-service
+  namespace: am
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.job: glob:prod-*
+spec:
+  releaseName: am-org-role-mapping-batch-service
+  rollback:
+    enable: true
+  chart:
+    name: am-org-role-mapping-batch-service
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    version: 0.0.1
+  values:
+    job:
+      schedule: "*/10 * * * *"
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/am/org-role-mapping-batch-service:prod-0cde3126
+    global:
+      environment: aat
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      enableKeyVaults: true
```

Comment:
Are you sure this should run on a single cluster? I suggesetd moving to "common" not simply deleting one.